packages/gui/src/components/ModeSelector.tsx

```diff
@@ -135,7 +120,24 @@ const ModeSelector = () => {
             }}
           >
             <Paper>
-              <ClickAwayListener onClickAway={handleClose}>
+              <ClickAwayListener
+                onClickAway={(event) => {
```

Comment:
I moved te `handleClose` fn inline, as otherwise der ea re typing issues I couldn't really figure out - will check this out in a successive PR.